backend/wargamebackend/wargamelogic/views/get.py

```
@@ -25,6 +25,7 @@
     TileSerializer,
 )
 from ..game_logic import *
+from ..check_roles import role_required, any_role_required
```

Ruff (0.12.2)

`..check_roles.any_role_required` imported but unused

Remove unused import: `..check_roles.any_role_required`

(F401)